### PR TITLE
docs: fixing typos

### PR DESCRIPTION
**Pull request by CoralWombat**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Fixed some typos in docs

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Review by zanettin — APPROVED** [review]
thank you for the adjustments 🙏

File: packages/docs/src/routes/docs/(qwikcity)/endpoints/index.mdx [hunk 1]
```diff
@@ -12,9 +13,9 @@ import CodeSandbox from '../../../../components/code-sandbox/index.tsx';
 # Endpoints
 
 
-A [middleware function](/docs/middleware) that returns a response is called an endpoint. Endpoints are useful for returning data such as  RESTful API, GraphQL API, JSON, XML, reverse proxy or any other time of API where you need to control all aspects of the HTTP response. Returning a response (using `json` for example) will stop the execution of subsequent middleware functions past this point in the middleware chain. 
+A [middleware function](/docs/middleware) that returns a response is called an endpoint. Endpoints are useful for returning data such as  RESTful API, GraphQL API, JSON, XML, reverse proxy or any other type of API where you need to control all aspects of the HTTP response. Returning a response (using `json` for example) will stop the execution of subsequent middleware functions past this point in the middleware chain. 
 
-All endpoints receive an [`RequestEvent`](/docs/middleware/#requestevent) API for controlling the response of the endpoint. 
+All endpoints receive a [`RequestEvent`](/docs/middleware/#requestevent) API for controlling the response of the endpoint.
```
Comment (zanettin) [inline packages/docs/src/routes/docs/(qwikcity)/endpoints/index.mdx:18]: imo this was correct alteady, not? the an refers to event

**Review by zanettin — COMMENTED** [review]
thx a lot for the updates! just left a question 👼

File: packages/docs/src/routes/docs/(qwikcity)/endpoints/index.mdx (hunk 1 above)
Comment (CoralWombat) [reply packages/docs/src/routes/docs/(qwikcity)/endpoints/index.mdx:18]: It refers to the "Request"

**Pull request merged**